test/user.js

```diff
@@ -0,0 +1,106 @@
+const chai = require('chai'),
+  dictum = require('dictum.js'),
+  server = require('./../app'),
+  User = require('./../app/models').user,
+  should = chai.should();
+
+describe('/users POST', () => {
+  it('Successful', done => {
```

Comment:
the idea of the it is to be readable and descriptive about the test, for example `it('responds successfully'` `it('creates a new user'`